packages/radix/src/components/Badge.tsx

```diff
@@ -29,11 +29,12 @@ export const BaseBadge = React.forwardRef<HTMLSpanElement, BadgeProps>((props, r
           height: 3, // TODO: Remove `inline-flex` from Primitives and then we can make this `inline-block` and remove height
           minWidth: 5,
           paddingY: 0,
-          paddingX: 1,
+          paddingX: '1.25ch',
```

Comment:
Can we not rely on the scale for this? 🙄

Reply:
Hey, neither scale value worked well. The space is so tight here that every pixel makes a difference.

It’s sort of typography related: the monospace font has pretty large spaces between words, so badge side paddings need to be a bit larger than the space character to look balanced. Too small — say, 5 px like we had, and badges with more than one word fall apart visually. More — and it looks elongated, sort of unbalanced compared to its small font.

Hence `ch` the unit. It should age well if we add another size of the badge or change its font size. Do you mind if we keep it this way?

Reply:
Fair enough, I'm ok keeping this, we already use `em` in some places anyway. Also this is the perfect use case for `ch` so let's go for it. Thanks!